projects/soft999_ahnaf_shahriar_fw_103/src/main.c

```diff
@@ -0,0 +1,61 @@
+#include "adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+static GpioAddress potentiometer_addr = {
+  .port = GPIO_PORT_A,
+  .pin = 6,
+};
+// function to call when the button gets hit
+static void button_interrupt_handler(const GpioAddress *addy, void *context) {
```

Comment:
we prefix static methods with `prv_` and global static variables with `s_`.

(addy is an interesting variable name, typo?)